contracts/VoucherKernel.sol

```diff
@@ -237,6 +237,7 @@ contract VoucherKernel is IVoucherKernel, Ownable, Pausable, ReentrancyGuard, Us
         require(_validFrom <= _validTo, "INVALID_VALIDITY_FROM");
```

Comment:
By adding require to line 240, this one becomes almost redundant -> when `_validTo >= _validFrom + 5 minutes` is true then  `_validFrom <= _validTo` is true as well.

I say almost because if you just remove line 237 there is an edge case, there `_validFrom` is set just below maxUint256 and adding 5 minutes would overflow, making require in 240 true even if `_validFrom` would actually be greater than `_validTo`. 

Suggestion: remove line 237 and add safemath to this line.

Reply:
done